cmd/thor/utils.go

```diff
@@ -250,39 +250,39 @@ func selectGenesis(ctx *cli.Context) (*genesis.Genesis, thor.ForkConfig, error)
 	}
 }
 
-func parseGenesisFile(uri string) (*genesis.Genesis, thor.ForkConfig, error) {
+func parseGenesisFile(uri string) (*genesis.Genesis, *thor.ForkConfig, error) {
 	var (
 		reader io.ReadCloser
 		err    error
 	)
 	if strings.HasPrefix(uri, "http://") || strings.HasPrefix(uri, "https://") {
 		res, err := http.Get(uri) // #nosec
 		if err != nil {
-			return nil, thor.ForkConfig{}, errors.Wrap(err, "http get genesis file")
+			return nil, nil, errors.Wrap(err, "http get genesis file")
 		}
 		reader = res.Body
 	} else {
 		reader, err = os.Open(uri)
 		if err != nil {
-			return nil, thor.ForkConfig{}, errors.Wrap(err, "open genesis file")
+			return nil, nil, errors.Wrap(err, "open genesis file")
 		}
 	}
 	defer reader.Close()
 
 	decoder := json.NewDecoder(reader)
 	decoder.DisallowUnknownFields()
 
-	var forkConfig = thor.NoFork
+	var forkConfig = &thor.NoFork
```

Comment:
```suggestion
	var forkConfig = thor.NoFork
```

Reply:
Previous code will overwrite `thor.NoFork`

Reply:
I think this is fine, it's the pointer of a new value.